packages/expo-asset/src/resolveAssetSource.web.ts

```diff
@@ -1,5 +1,6 @@
+import { getAssetByID } from '@react-native/assets/registry';
```

Comment:
If want to keep the compat, we could add an optional alias of `@react-native/assets/registry` to `@react-native/assets-registry/registry` in the babel plugin. Alternatively, we could import from `react-native/Libraries/Image/AssetRegistry` which re-exports the correct location for the version of React Native that's installed.